src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1129,6 +1129,7 @@ a.test-arrow:hover{
 	text-decoration: none;
 }
 
+.small-section-header:hover a:before,
```

Comment:
Because of this change, the anchor on `Methods from Deref<Target = str>` targets `str`, is blue and overlaps the text too.

Reply:
Fixed!